solana/pyth2wormhole/client/tests/test_migrate.rs

```diff
@@ -174,16 +170,16 @@ async fn test_migrate_already_migrated() -> Result<(), solitaire::ErrBox> {
     info!("Before start_with_context");
     let mut ctx = p2w_test.start_with_context().await;
 
-    let migrate_tx = p2wc::gen_migrate_tx(
-        ctx.payer,
-        p2w_program_id,
-        p2w_owner,
-        ctx.last_blockhash,
-    )?;
+    let migrate_tx =
+        p2wc::gen_migrate_tx(ctx.payer, p2w_program_id, p2w_owner, ctx.last_blockhash)?;
     info!("Before process_transaction");
 
     // Migration should fail because the new config account is already initialized
-    assert!(ctx.banks_client.process_transaction(migrate_tx).await.is_err());
```

Comment:
i take it we don't have clippy / pre-commit here which is why there's all this formatting noise?

we should install clippy. it's annoying to review PRs with these format changes mixed in.

Reply:
I will try to undo the formatting-related changes. Sorry about that!